compiler/cpp/src/thrift/parse/t_field.h

```diff
@@ -68,7 +68,7 @@ class t_field : public t_doc {
 
   int32_t get_key() const { return key_; }
 
-  enum e_req { T_REQUIRED, T_OPTIONAL, T_OPT_IN_REQ_OUT };
+  enum e_req { T_REQUIRED = 0u, T_OPTIONAL, T_OPT_IN_REQ_OUT = ~0u };
```

Comment:
This changes the value of T_OPT_IN_REQ_OUT from before - is this ever on the wire?  Would an older client end up sending "2" here instead of ~0u ?